README.md

```diff
@@ -188,3 +188,17 @@ var customVocabularySubmission = await client.submitCustomVocabularies(customVoc
 // Get information regarding the Custom Vocabulary submission and its progress
 var customVocabularyInformation = await client.getCustomVocabularyInformation(customVocabularySubmission.id)
 ```
+
+# For Rev.ai NodeSDK Developers
+
+After cloning and installing required npm modules, you should follow these practices when developing:
+
+1. Use the scripts defined in [package.json](https://github.com/revdotcom/revai-node-sdk/tree/develop/package.json) in this manner `npm run [command_name]`:
+    1. `lint` checks that you are not violating any code style standards. This ensures our code's style quality stays high improving readability and reducing room for errors.
+    2. `build` transpiles the typescript into javascript with the options specified in [tsconfig.json](https://github.com/revdotcom/revai-node-sdk/tree/develop/tsconfig.json)
+    3. `unit-test` runs our unit tests which live in the [unit test directory](https://github.com/revdotcom/revai-node-sdk/tree/develop/test/unit).
+    * Note that `integration-test` is currently configured to work with a certain account specified in our continuous integration build environment, as such for now you can check the automated continuous integration checks to pass the integration tests. This will change in the future.
+2. Add any relevant test logic if you add or modify any features in the source code and check that the tests pass using the scripts mentioned above.
+3. Update the examples provided to illustrate any relevant changes you made, and check that they work properly.
+    * Note that you will have to copy the output of the `build` script into the `examples/node_modules/revai-node-sdk` directory in order to use your changes in the examples. On Unix, this can be simply done with the following command when in the root directory: `$ cp -r dist/src examples/node_modules/revai-node-sdk/`.
```

Comment:
This is not a requirement, there are many ways to do this. Mention that this is one method.

Reply:
another method is to install from source

Reply:
npm install from source

Reply:
@amikofalvy how does it look now?